MagnaLorem/CampusAlly
# Chat rooms

### what was implemented 

1. multiple chat room. You only see the messages in the particular chat room. For example, you don't see messages that are sent to chat room A if you are in chat room B.
2. empty message issue fixed. You can't send empty messages.

### How to test 

Open up localhost in 2 different tabs(windows). 
1. test to see if you can retrieve the messages if you are in the same chat rooms. 
2. test to see if you won't see the messages if you are in the different chat rooms.
